core/audits/bootup-time.js

```diff
@@ -128,7 +128,7 @@ class BootupTime extends Audit {
         label: str_(UIStrings.columnScriptParse)},
     ];
 
-    const details = BootupTime.makeTableDetails(headings, results, summary);
+    const details = BootupTime.makeTableDetails(headings, results, summary, [['total']]);
```

Comment:
Seems like we could remove these touples and just use the string here. Once we know which value the table is sorted by, it should be easy to determine if it's in ascending or descending order right?

Reply:
So far every sort seems to follow the pattern — if it's numeric, it's descending order and if it's alphanumeric, it's ascending. May be this is a fair assumption to make for future as well?